requirements.txt

```diff
@@ -41,10 +43,6 @@ git+https://github.com/CenterForOpenScience/modular-odm.git@0.4.0
 # Python markdown extensions for comment emails
 git+git://github.com/CenterForOpenScience/mdx_del_ins.git
 
-# Kombu with the ability to specify queue priority
-# TODO: Remove this when Kombu has a stable release including commit c20f854
-git+git://github.com/CenterForOpenScience/kombu.git@v3.0.36
```

Comment:
Confirmed that this [has made it upstream](https://github.com/celery/kombu/blame/master/kombu/entity.py#L851), which means we can probably delete this fork.